tools/clang-archer++.in

```diff
@@ -68,4 +68,4 @@ else
   link_flags=""
 fi
 
-@LLVM_ROOT@/bin/clang++ -I@OMP_PREFIX@/include -Xclang -load -Xclang @CMAKE_INSTALL_PREFIX@/lib/LLVMArcher.so -fopenmp -fsanitize=thread $link_flags -g "$@"
+@LLVM_ROOT@/bin/clang++ -I@OMP_PREFIX@/include -Xclang -load -Xclang @CMAKE_INSTALL_PREFIX@/lib/LLVMArcher.so -std=c++11 -fopenmp -fsanitize=thread $link_flags -g "$@"
```

Comment:
What is the main reason should Archer only accept the code conforming to C++11? Should Archer fail to compile if the users' code use some non-standard features only available in Clang or GCC, which stray away form C++11? 

Would the old way be better -- we don't specify the C++ standard in this wrapper, which still allow the user to specify a standard if they want?